src/vivarium_cluster_tools/psimulate/runner.py

```diff
@@ -258,6 +283,12 @@ def main(
     registry_manager.enqueue(
         jobs=job_parameters, workhorse_import_path=worker.WORK_HORSE_PATHS[command]
     )
+
+    if make_backups:
+        write_backup_metadata(
+            backup_metadata_path=output_paths.backup_metadata_path,
+            parameters_by_job=registry_manager.get_params_by_job(),
```

Comment:
Just to be clear, this will record all jobs even if we have more jobs queued than our max jobs (e.g. if we submitted 10k jobs with max jobs of 8k, would they all be recorded here, or only the first 8k).

Reply:
yes, it's really just defining the mapping between the branch configuration and the job ID, not making any guarantees about what is being run or what files will exist.